ocs2_robotic_examples/ocs2_legged_robot_ros/CMakeLists.txt

```diff
@@ -3,9 +3,11 @@ project(ocs2_legged_robot_ros)
 
```

Comment:
Lots of unrelated changes in this file, I'm not sure if anything actually changed? Can you revert the formatting noise?

Reply:
I think you were looking at outdated commits!